### ISSUE TITLE
Architect mode not available

### ISSUE THREAD

**Issue by Nova-lotus** [paul-gauthier/aider#1801]
### Issue

Not in the in-chat commands and can't run it with --architect

Here's all I can use, I'm using the latest version but doesn't seem like I have architect, has it been removed or something? Also even when I use the inchat command /chat-mode I only see help, code and ask

[-h] [--openai-api-key OPENAI_API_KEY] [--anthropic-api-key ANTHROPIC_API_KEY] [--model MODEL] [--opus]
             [--sonnet] [--4] [--4o] [--mini] [--4-turbo] [--35turbo] [--deepseek] [--list-models MODEL]
             [--openai-api-base OPENAI_API_BASE] [--openai-api-type OPENAI_API_TYPE]
             [--openai-api-version OPENAI_API_VERSION] [--openai-api-deployment-id OPENAI_API_DEPLOYMENT_ID]
             [--openai-organization-id OPENAI_ORGANIZATION_ID] [--model-settings-file MODEL_SETTINGS_FILE]
             [--model-metadata-file MODEL_METADATA_FILE] [--verify-ssl | --no-verify-ssl] [--edit-format EDIT_FORMAT]     
             [--weak-model WEAK_MODEL] [--show-model-warnings | --no-show-model-warnings] [--map-tokens MAP_TOKENS]       
             [--map-refresh {auto,always,files,manual}] [--cache-prompts | --no-cache-prompts]
             [--cache-keepalive-pings CACHE_KEEPALIVE_PINGS] [--map-multiplier-no-files MAP_MULTIPLIER_NO_FILES]
             [--max-chat-history-tokens MAX_CHAT_HISTORY_TOKENS] [--env-file ENV_FILE]
             [--input-history-file INPUT_HISTORY_FILE] [--chat-history-file CHAT_HISTORY_FILE]
             [--restore-chat-history | --no-restore-chat-history] [--llm-history-file LLM_HISTORY_FILE] [--dark-mode]     
             [--light-mode] [--pretty | --no-pretty] [--stream | --no-stream] [--user-input-color USER_INPUT_COLOR]       
             [--tool-output-color TOOL_OUTPUT_COLOR] [--tool-error-color TOOL_ERROR_COLOR]
             [--tool-warning-color TOOL_WARNING_COLOR] [--assistant-output-color ASSISTANT_OUTPUT_COLOR]
             [--code-theme CODE_THEME] [--show-diffs] [--git | --no-git] [--gitignore | --no-gitignore]
             [--aiderignore AIDERIGNORE] [--subtree-only] [--auto-commits | --no-auto-commits]
             [--dirty-commits | --no-dirty-commits] [--attribute-author | --no-attribute-author]
             [--attribute-committer | --no-attribute-committer]
             [--attribute-commit-message-author | --no-attribute-commit-message-author]
             [--attribute-commit-message-committer | --no-attribute-commit-message-committer] [--commit]
             [--commit-prompt PROMPT] [--dry-run | --no-dry-run] [--lint] [--lint-cmd LINT_CMD]
             [--auto-lint | --no-auto-lint] [--test-cmd TEST_CMD] [--auto-test | --no-auto-test] [--test] [--file FILE]   
             [--read FILE] [--vim] [--voice-language VOICE_LANGUAGE] [--chat-language CHAT_LANGUAGE] [--version]
             [--just-check-update] [--check-update | --no-check-update] [--install-main-branch] [--upgrade]
             [--apply FILE] [--yes] [-v] [--show-repo-map] [--show-prompts] [--exit] [--message COMMAND]
             [--message-file MESSAGE_FILE] [--encoding ENCODING] [-c CONFIG_FILE] [--gui]
             [--suggest-shell-commands | --no-suggest-shell-commands]

### Version and model info

Aider-Chat 0.57.1
Model: Qwen, Gemini

**Comment by fry69** [comment]
The documentation on https://aider.chat/ gets auto generated from the `main` branch, so it may include features that are not available in the PyPI version.
TL;DR install the main branch version with
```
aider --install-main-branch
```
and try again. Or wait until the next version gets released. It should include all features from the `main` branch.

**Comment by Nova-lotus** [comment]
I see, alright thank you I did not know that, I'll wait until the next version. Thanks again

**Comment by fry69** [comment]
This probably should get mentioned on the website somewhere.

Apart from that the Architect mode currently gets only linked from release notes page, where it clearly says that this is in the `main` branch and not released yet.

https://aider.chat/HISTORY.html

So how did you find out about it?

**Comment by Nova-lotus** [comment]
I found out from https://aider.chat/docs/usage/commands.html and https://aider.chat/docs/usage/modes.html#architect-mode-and-the-editor-model
